StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1642UnitTests.cs

```diff
@@ -940,7 +940,7 @@ public ClassName()
     public class ClassName
     {{
         /// <summary>
-        /// Initializes a new instance of the <see cref=""ClassName""/> class.{emptyContent}</summary>
+        /// Initializes a new instance of the <see cref=""ClassName""/> class.</summary>
```

Comment:
❔  Is there a straightforward way to also make this place the `</summary>` on the following line? If this isn't resolved before merge we should file a follow-up issue for it.